docs/configuring-socorro.rst

```diff
@@ -152,3 +152,58 @@ All of these jobs can be enabled or disabled in crontabber configuration or by m
 DEFAULT_JOBS in:
 
 https://github.com/mozilla/socorro/blob/master/socorro/cron/crontabber_app.py
+
+
+Symbols S3 uploads
+------------------
```

Comment:
Nice, thanks for documenting this :+1: "how do I upload symbols" is a super common question